### PR TITLE
Change the Node.js beginners page to recommend the Windows native installation

### LINKED ISSUE #2011: Why suggest using a Windows Insider build if the audience is "beginners "
I like WSL, but suggesting a beta OS build and an unreleased WSL2 build for beginners doesn't seem to match the audience of the article

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 811fa359-feec-10dd-544c-881982b25138
* Version Independent ID: aeb2c4e1-28d2-65c8-06e7-b63a3233c5e2
* Content: [Get started using NodeJS on Windows for beginners](https://docs.microsoft.com/en-us/windows/nodejs/beginners)
* Content Source: [hub/nodejs/beginners.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/nodejs/beginners.md)
* Product: **dev-environment**
* Technology: **windows-nodejs**
* GitHub Login: @mattwojo
* Microsoft Alias: **mattwoj**

### PR DESCRIPTION
**Pull request by joaocgreis**
The instructions provided on the beginners page with this change should work in any shell, including WSL.

This also includes a few small fixes, like updating the Node.js LTS version.

cc @mattwojo, let me know if this needs any change or feel free to edit!

Fixes: https://github.com/MicrosoftDocs/windows-uwp/issues/2011

### CODE REVIEW

**Comment by PRMerger20** [comment]
@joaocgreis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mattwojo** [comment]
Thanks @joaocgreis! I had already started a rewrite of this Node beginners page on another branch, but just went through all of your changes and combined them into my previous updates. Would you mind taking a look and giving me a thumbs up if this PR looks alright to you?
https://github.com/MicrosoftDocs/windows-uwp-pr/pull/515

File: hub/nodejs/beginners.md [hunk 1]
```diff
@@ -35,17 +32,17 @@ Node.js is a JavaScript runtime primarily used for creating web applications. Pu
...
 - **Hardware programming**: While not quite as popular as web apps, Node.js is growing in popularity for IoT uses, such as collecting data from sensors, beacons, transmitters, motors, or anything that generates large amounts of data. Node.js can enable data collection, analyzing that data, communicating back and forth between a device and server, and taking action based on the analysis. NPM contains more than 80 packages for Arduino controllers, raspberry pi, Intel IoT Edison, various sensors, and Bluetooth devices.
 
 ## Try using Node.js in VS Code
 
-1. Open your Ubuntu terminal and create a new directory: `mkdir HelloNode`, then enter the directory: `cd HelloNode`
+1. Open your terminal and create a new directory: `mkdir HelloNode`, then enter the directory: `cd HelloNode`
 
-2. Create an empty JavaScript file named "app.js": `touch app.js`
+2. Open the directory and empty file in VS Code: : `code .`
```
Comment (bnb) [inline hub/nodejs/beginners.md:43]: Don't you need to install the VS Code CLI into the path from within VS Code? I may be missing the step to do that above, but if not we should ensure that we don't confuse users who haven't done that.

**Review by bnb — COMMENTED** [review]
I have some other feedback on this article in general, but since this PR is scoped to just updating it to native Windows, I'll leave those for a different discussion.

File: hub/nodejs/beginners.md (hunk 1 above)
Comment (joaocgreis) [reply hub/nodejs/beginners.md:43]: Thanks for reviewing! The installer downloaded from https://code.visualstudio.com/ offers an option to add to the path, which is enabled by default. So, I think this is good as is, but please let me know if I missed something.

**Comment by joaocgreis** [comment]
@mattwojo I don't have access to the other PR, but I see it has already landed here including my changes. It looks great, thanks!

I'll close this PR since it would be better to open a new PR for any other change.

**Pull request closed without merging**